Anlab.Core/Services/MailService.cs

```diff
@@ -106,5 +109,10 @@ public void SendFailureNotification(int orderId, string failureReason)
                 Log.Error($"Error sending email for Failure reason. OrderId {orderId} Exception {ex.Message}" );
             }
         }
+
+        public bool DisableSend()
```

Comment:
Change the name to like `IsSendDisabled` because I thought this was a method to actually disable send.  In fact maybe you could just do it as a prop like so:
`public bool IsSendDisabled { get { return ... } }`